File: app/src/main/java/com/example/richtext/utils/JumpUtil.java
package com.example.richtext.utils;

import android.app.Activity;
import android.content.Intent;

import com.example.richtext.TopicListActivity;
import com.example.richtext.UserListActivity;

/**
 * Created by shuyu on 2016/11/10.
 */

public class JumpUtil {

    public static void goToUserList(Activity activity, int code) {
        Intent intent = new Intent(activity, UserListActivity.class);
        activity.startActivityForResult(intent, code);
    }


    public static void goToTopicList(Activity activity, int code) {
        Intent intent = new Intent(activity, TopicListActivity.class);
        activity.startActivityForResult(intent, code);
    }
}

File: app-kotlin/src/main/java/com/example/shuyu/utils/JumpUtil.kt
package com.example.shuyu.utils

import android.app.Activity
import android.content.Intent

import com.example.shuyu.TopicListActivity
import com.example.shuyu.UserListActivity

open class JumpUtil {

    companion object {
        fun goToUserList(activity: Activity, code: Int) {
            val intent = Intent(activity, UserListActivity::class.java)
            activity.startActivityForResult(intent, code)
        }


        fun goToTopicList(activity: Activity, code: Int) {
            val intent = Intent(activity, TopicListActivity::class.java)
            activity.startActivityForResult(intent, code)
        }
    }

}